### PR TITLE
Import Sql Field Type Guesser

### PR DESCRIPTION
**Pull request by harish81**
## Import Sql Field Type Guesser(Smart Fields)
This feature will guess the field's type based on their datatype or name.  _This will helps to improve workflow while generating random data for tables._
See Demo:
import_sql_smart_field

## Changes Explaination
 - **resources/lang/en.php** - Messages.
 - **resources/scripts/importSql.js** - The Field Guesser Code.
 - **resources/templates/generate.tab1.tpl** - The Checkbox to enable/disable Feature.

### CODE REVIEW

**Comment by benkeen** [comment]
Hey @harish81 this looks interesting. Could you explain exactly what this does? From a quick scan it looks like it's using the name of the field to make a guess about what the data type should be.

**Comment by benkeen** [comment]
By the way, I made a few tweaks to the feature in master, mostly just to improve error handling a bit. Let me know if you have any questions.

**Comment by harish81** [comment]
Hi @benkeen . This feature will helps to quickly generate data for table without selecting each & every field. This gone be helpful while generating random test data for table (as a developer for testing). 

Have you look at code? i have included common field name and if name could not be found then it will select type based on sql data-type of filed. 

In our company we use this feature to fill our table with random data-set for testing. Our designer use this to design UI with real data-set.

In short it is random test factory generator, after that user can review type & can change type.

**Comment by harish81** [comment]
Hi @benkeen . As we know most tables has some common field such as name, city, dates ... etc.
this feature will select type for these common fields for us. This will reduce overhead to select type of each fields.

**Comment by harish81** [comment]
I have removed confliction in above commit.

**Comment by benkeen** [comment]
Thanks @harish81!
